package.json

```diff
@@ -2,7 +2,6 @@
   "name": "@kenrick95/c4",
   "version": "3.0.5",
   "description": "Connect Four game",
-  "main": "./dist/lib.js",
```

Comment:
This is actually still an issue for me. I would like to have this, but Parcel 2 wouldn't let me ...